common/environment/build-style/cmake.sh

```diff
@@ -1,5 +1,5 @@
 if [ "$CHROOT_READY" ]; then
 	if [[ "$hostmakedepends" != *"cmake-bootstrap"* ]]; then
-		hostmakedepends+=" cmake"
+		hostmakedepends+=" cmake ninja"
```

Comment:
`cmake` itself is built with `build_style=cmake`, and has `cmake-bootstrap` in `hostmakedepends`.
I suppose we should keep `cmake` with `-G'Unix Makefiles'`?

Reply:
`cmake-bootstrap` doesn't know how to generate Ninja, so I'll add `-G'Unix Makefiles'` to `cmake`'s `configure_args`